providers/oauth/oauth.webrequest.js

```diff
@@ -42,26 +42,29 @@ ChromeWebRequestAuth.prototype.initiateOAuth = function(redirectURIs, continuati
   return false;
 };
 
-ChromeWebRequestAuth.prototype.launchAuthFlow = function(authUrl, stateObj, continuation) {
+ChromeWebRequestAuth.prototype.launchAuthFlow = function(authUrl, stateObj, interactive, continuation) {
   "use strict";
+  if (interactive === undefined) {
+    interactive = true;
+  }
   var listener = this.reqListener.bind(this, stateObj, continuation);
   this.listeners[stateObj.state] = listener;
   chrome.webRequest.onBeforeRequest.addListener(listener, {
     types: ["main_frame"],
     urls: [stateObj.redirect]
   });
-  
+
   chrome.tabs.create({
     url: authUrl,
-    active: true
+    active: interactive
```

Comment:
What is the desired behavior of 'interactive'?
If it is to match https://developer.chrome.com/apps/identity
then this behavior would still open up a tab.
I believe chrome.identity does some caching and will fail if its cached.